.vitepress/constants/mocha_versions.js

```diff
@@ -1,8 +1,8 @@
 const mochaVersions = Object.freeze({
-  "app-latest-tag": "v1.6.0",
-  "app-latest-sha": "6f19fd83c4d0061c717a40e23a368a2f60aff715",
-  "core-latest-tag": "v1.34.0-tm-v0.34.29",
-  "core-latest-sha": "043dbc5f7b5e7cadfbb9d25d8a736a477586e176",
+  "app-latest-tag": "v1.7.0-rc0",
```

Comment:
```suggestion
  "app-latest-tag": "v1.7.0,
```

This uses the v1.7.0 tag, as the v1.7.0-rc0 tag was tested and no changes were made, so was released under same commit.